### DISCUSSION TITLE
xTB MD dump decomposition products

### DISCUSSION THREAD

**Discussion by loggging** [grimme-lab/xtb#506] [Q&A]
**Is your feature request related to a problem? Please describe.**
Hi, I am trying to run MD simulation for the decomposition of a compound with xtb. However, unlike lammps, xtb does not dump decomposition products and bond information. I am wondering whether there are any scripts to realize this? Besides, annealing function was removed after v6.2. I want to let the temperature of the system rise. How should I do?

**Describe the solution you'd like**
Add the species and bonds command to dump these information.

**Describe alternatives you've considered**
Write scripts.

**If possible state how you can assist in providing data or code to implement the feature**
I do not know any programming language. Please help.

**Additional context**
Anyone please help me solve this problem.

**Comment by awvwgk** [comment]
The atomic simulation environment ([ASE](https://wiki.fysik.dtu.dk/ase/)) can be used to drive calculations with `xtb` and can be used to implement such analysis on top of a trajectory easily.

There is also a somewhat related feature in `crest`: https://github.com/grimme-lab/crest/issues/13#issuecomment-700684290
